packages/gestalt-datepicker/src/DatePickerTextField.js

```diff
@@ -0,0 +1,98 @@
+// @flow strict-local
+import * as React from 'react';
+import { Box, Icon, Touchable, TextField } from 'gestalt';
+import PropTypes from 'prop-types';
+
+type OwnProps = {|
+  accessibilityLabelIcon: string,
+  isRTL?: boolean,
+|};
+
+type InjectedProps = {|
```

Comment:
The name InjectedProps sounds like and HOC so it took me a minute to realize these are props that react-datepicker adds on. Could you make the name more obvious or add a comment about that here?

Reply:
FIXED. thanks for pointing out